org.gnome.Fractal.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id": "org.gnome.Fractal",
+    "version": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "fractal",
+    "finish-args": ["--share=network", "--socket=x11", "--filesystem=host" ],
+    "build-options" : {
+        "env": {
+            "CARGO_HOME": "/run/build/Fractal/cargo",
+            "V": "1"
+        },
+        "build-args": [ "--share=network" ]
+    },
+    "modules": [
+        {
+           "name": "Fractal",
+           "buildsystem": "simple",
+           "build-commands": [
+               "source /usr/lib/sdk/rust-stable/enable.sh && ./configure --prefix=/app && make && make install"
```

Comment:
I don't like it but I don't think there's any other way to source a file before build system kicks in. @TingPing @alexlarsson ideas?

Reply:
The enable.sh file only adds rust to the PATH, but currently there's not way to expand environment vars. Once this issue https://github.com/flatpak/flatpak-builder/issues/18 is fixed we can remove it and use env build option:

Reply:
You can use this:
https://github.com/flathub/org.gnome.gbrainy/blob/master/org.gnome.gbrainy.json#L21